.travis/install.sh

```diff
@@ -18,7 +18,7 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     brew install xz  # required for python lzma module
     brew outdated pyenv || brew upgrade pyenv
     brew install pkg-config
-    brew install Caskroom/versions/osxfuse-beta
+    brew install Caskroom/versions/osxfuse
```

Comment:
while this was a at first unsuccessful attempt to fix the issue, a brew maintainer now said he will remove the -beta and it will be just osxfuse in future. so this is correct in the end.